lib/game.rb

```diff
@@ -1,6 +1,5 @@
 class Game
-
-  attr_reader :id,
+  attr_reader :game_id,
               :season,
```

Comment:
this was a resolved merge conflict from dannys game class I believe. I changed to game_id to match some of my methods. This will likely need to be re-resolved later.

Reply:
Check